manifests/quick-start/base/webhooks/argo-workflows-webhook-clients-secret.yaml

```diff
@@ -0,0 +1,22 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: argo-workflows-webhook-clients
+# The data key must be the name of a service account token. Typically you'll want this to be the same as the service account name.
+stringData:
+  # https://support.atlassian.com/bitbucket-cloud/docs/manage-webhooks/
+  bitbucket.org: |
+    type: bitbucket
+    secret: "my-uuid"
+  # https://support.atlassian.com/bitbucket-cloud/docs/manage-webhooks/
```

Comment:
Cloud and server are different products. Should this be https://confluence.atlassian.com/bitbucketserver/managing-webhooks-in-bitbucket-server-938025878.html?